src/mono/mono/metadata/object-internals.h

```diff
@@ -1997,6 +1997,9 @@ mono_string_hash_internal (MonoString *s);
 MONO_COMPONENT_API int
 mono_object_hash_internal (MonoObject* obj);
 
+MONO_COMPONENT_API int
```

Comment:
nit: this doesn't need to be a component API - it's not used by any component (AFAICT).  Just an unadorned declaration would be sufficient.

Reply:
Thanks, I removed it.